annotate_confidence_splits.py

```
@@ -0,0 +1,151 @@
+                    gold_spans = calculate_span_score(
+                        start_tuple,
+                        end_tuple,
+                        torch.sigmoid(scores_inside[batch_idx]),
+                        torch.sigmoid(scores_start[batch_idx]),
+                        torch.sigmoid(scores_end[batch_idx]),
+                        id_to_class_i,
+                    )
+                    gold_spans = greedy_search(gold_spans, True, False)
+                    annotations.append({**original_input[0], "gold_spans_with_confidence": gold_spans})
+                
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)